plugins/@grouparoo/calculated-property/src/lib/profileProperty.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  GrouparooCLI,
+  plugin,
+  Profile,
+  ProfilePropertyPluginMethod,
+} from "@grouparoo/core";
+import { NodeVM } from "vm2";
+
+async function calculateProfilePropertyValue(
+  customFunction,
+  profile: Profile,
+  propertyName: string
+): Promise<string> {
+  let populatedFunction;
+  try {
+    populatedFunction = await plugin.replaceTemplateProfileVariables(
+      customFunction,
+      profile,
+      false
+    );
+    //fail at every level if someone tries to require a library... this should never be allowed to hit vm.run
```

Comment:
This means that running `roo validate` or `roo apply` will fail.  It fails a bit more abruptly than with other config errors, but I think that's warranted.  We don't want anything questionable attempting to be executed.